files/Aboot/boot0.j2

```diff
@@ -502,6 +502,7 @@ write_platform_specific_cmdline() {
     if [ "$sid" = "Gardena" ] || [ "$sid" = "GardenaE" ]; then
         aboot_machine=arista_7260cx3_64
         flash_size=28000
+        cmdline_add logs_inram=on
```

Comment:
What is the partition size? We might need to reduce the partition size when putting it on tmpfs

Reply:
It will be just under 800MB. Prior to this, on disk, it was close to 4GB.

Reply:
800MB was calculated at run time?

Reply:
Yes, calculated between 5% and 10% of total RAM (unless there's an override for a certain hwsku in `files/initramfs-tools/union-mount.j2`).